.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: php
 
 php:
+  - 7.1
   - 7.0
-  - 5.5
```

Comment:
The plugin is compatible with older cake versions which support php 5.5 so we should keeping testing on 5.5.